tests/features/joinup_event/add_event.solution.feature

```diff
@@ -0,0 +1,64 @@
+@api
+Feature: "Add event" visibility options.
+  In order to manage events
+  As a solution member
+  I need to be able to add "Event" content through UI.
+
+  Scenario: "Add event" button should not be shown to normal members, authenticated users and anonymous users.
+    Given the following solutions:
+      | title           | logo     | banner     |
+      | Ragged Tower    | logo.png | banner.jpg |
+      | Prince of Magic | logo.png | banner.jpg |
+    And the following collection:
+      | title      | Collective Ragged tower       |
+      | logo       | logo.png                      |
+      | banner     | banner.jpg                    |
+      | affiliates | Ragged Tower, Prince of Magic |
+
+    When I am logged in as an "authenticated user"
+    And I go to the homepage of the "Ragged Tower" solution
+    Then I should not see the link "Add event"
+
+    When I am an anonymous user
+    And I go to the homepage of the "Ragged Tower" solution
+    Then I should not see the link "Add event"
+
+    When I am logged in as a "facilitator" of the "Ragged Tower" solution
+    And I go to the homepage of the "Ragged Tower" solution
+    Then I should see the link "Add event"
```

Comment:
OK, it passes if I reinstall the site. The problem was because I forgot to import the configuration change from web/modules/custom/joinup_event/config/install/field.field.node.event.og_audience.yml.

Side note: However, we are not using a usual D8 structure. Ideally we should have a single config (sync) directory under VCS and not spread the config YAMLs all over the codebase. Then we point {{$config_directories['sync']}} in settings.php to that directory and, when a config change lands through VCS, we simply perform {{$ drush cim}} and import only the differences. With the actual structure, I wonder how we'll update the configuration on production?